code/game/objects/effects/spawners/random/exotic.dm

```diff
@@ -48,67 +48,29 @@
 		/obj/item/storage/medkit/toxin = 35,
 		/obj/item/storage/medkit/brute = 27,
 		/obj/item/storage/medkit/fire = 27,
-		/obj/item/grenade/clusterbuster/smoke = 15,
-		/obj/item/clothing/under/chameleon = 13,
 		/obj/item/storage/toolbox/syndicate = 12,
-		/obj/item/melee/baton/telescopic = 12,
-		/obj/item/melee/baton = 11,
-		/obj/item/book/granter/spell/smoke = 10,
-		/obj/item/book/granter/spell/blind = 10,
-		/obj/item/clothing/shoes/chameleon/noslip = 10,
-		/obj/item/grenade/c4 = 7,
 		/obj/item/borg/upgrade/ddrill = 3,
-		/obj/item/borg/upgrade/soh = 3,
+		/obj/item/knife/butcher = 14,
+		/obj/item/clothing/glasses/night = 10,
+		/obj/item/pickaxe/drill/diamonddrill = 6,
 	)
 
 /obj/effect/spawner/random/exotic/antag_gear
 	name = "antag gear"
 	icon_state = "esword"
 	loot = list(
-		/obj/item/storage/medkit/tactical = 35,
-		/obj/item/book/granter/spell/summonitem = 20,
-		/obj/item/book/granter/spell/forcewall = 17,
-		/obj/item/pneumatic_cannon = 15,
-		/obj/item/book/granter/spell/knock = 15,
-		/obj/item/storage/backpack/holding = 12,
```

Comment:
this is station-researchable yet valuable and is unrelated to combat; why was this removed?